File: clients/asset-client/src/constants/LogLevelConstant.ts
import { createConstant } from "./Constant";

const LogLevelConstant = createConstant(
  {
    Info: "Info",
    Warning: "Warning",
    Error: "Error",
  } as const,
  {
    Info: { displayName: "Thông báo" },
    Warning: { displayName: "Cảnh báo" },
    Error: { displayName: "Lỗi" },
  }
);

export default LogLevelConstant;

File: API/Hinet.Service/Constant/LogLevelConstant.cs
using System.ComponentModel;
namespace Hinet.Service.Constant
{
    public class LogLevelConstant
    {
        [DisplayName("Thông báo")]
        public static string Info => "Info";

        [DisplayName("Cảnh báo")]
        public static string Warning => "Warning";

        [DisplayName("Lỗi")]
        public static string Error => "Error";

    }
}
